### PR TITLE
Logging test objects were copy-pasta from logging repo, removing and usi...

### PR DESCRIPTION
**Pull request by BrennanConroy**
...ng official version

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@BrennanConroy__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Brennan Conroy). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: test/Microsoft.AspNet.Routing.Tests/project.json [hunk 1]
```diff
@@ -1,22 +1,23 @@
 {
-    "compilationOptions": {
```
Comment (Eilon) [inline test/Microsoft.AspNet.Routing.Tests/project.json]: It looks to me like most repos use 4 spaces per indent in JSON files. At least, in the few places I looked.

**Comment by Eilon** [comment]
Aside from indent, :shipit:

File: test/Microsoft.AspNet.Routing.Tests/project.json (hunk 1 above)
Comment (BrennanConroy) [inline test/Microsoft.AspNet.Routing.Tests/project.json]: Ya, not sure how that changed, just ctrl-k-f the file and it does the 4 spaces

**Pull request closed without merging**